eng/scripts/appium-install.ps1

```diff
@@ -67,9 +67,11 @@ if (!(Test-Path $logsDir -PathType Container)) {
 $AppiumHome = $env:APPIUM_HOME
 Write-Output "APPIUM_HOME: $AppiumHome"
 
-if (Test-Path $AppiumHome) {
-    Write-Output  "Removing existing APPIUM_HOME Cache..."
-    Remove-Item -Path $AppiumHome -Recurse -Force
+if ($AppiumHome) {
```

Comment:
Should use **Write-Output** indicating that needs to set the APPIUM_HOME if is null?

Reply:
I think the point here is that it tries to find Appium and if it doesn't find it or its not the version we expect then its going to uninstall and install: https://github.com/dotnet/maui/pull/22883/files#diff-2d6fe919f8e6585ce7f9aa99776ffc5fe0e737205eca6b3cf0a5ab6576a42870R90

This check only make sure that the script doesn't crash on a null reference exception when its not installed